crates/types/src/data.rs

```diff
@@ -692,4 +698,29 @@ pub mod null_block {
             Err(_) => None,
         }
     }
+
+    /// Builder fee data for a null block payload
+    #[must_use]
+    pub fn fee_data<TYPES: NodeType>(num_storage_nodes: usize) -> Option<FeeData<TYPES>> {
+        let (_, priv_key) =
+            <TYPES::BuilderSignatureKey as BuilderSignatureKey>::generated_from_seed_indexed(
+                [0_u8; 32], 0,
+            );
+
+        let (null_block, null_block_metadata) =
+            <TYPES::BlockPayload as BlockPayload>::from_transactions([]).ok()?;
+
+        match TYPES::BuilderSignatureKey::sign_fee(
+            &priv_key,
+            0,
+            &null_block.builder_commitment(&null_block_metadata),
+            &commitment(num_storage_nodes)?,
+        ) {
+            Ok(sig) => Some(FeeData {
+                fee_amount: 0,
```

Comment:
Could we get a comment explaining why the amount is 0 here? It's hard to understand if I didn't already have context.